sys/include/net/ng_ipv6/nc.h

```diff
@@ -123,6 +124,11 @@ typedef struct {
     uint8_t l2_addr_len;                    /**< Length of ng_ipv6_nc_t::l2_addr */
     uint8_t flags;                          /**< Flags as defined above */
```

Comment:
It's a problem in general to reference a group of flags in dosage. The defines prefixed with `NG_IPV6_NC_...` in the document group "Flag definitions for `ng_ipv6_ref_t` are meant. I'm against using enums for flags (see https://github.com/RIOT-OS/RIOT/pull/2614#discussion_r28945055 on that note). Have you maybe a better solution for referencing them?